### ISSUE TITLE
1.1.6 Not released on Atmosphere

### ISSUE THREAD

**Issue by thomasvanlankveld** [softwarerero/meteor-accounts-t9n#127]
I have a bug in my application that originates in version 1.1.4 of this package.

When I tried to specify that my app should use 1.1.6 I got a conflict error:

```shell
* softwarerero:accounts-t9n@1.1.6 <- my-app:external-dependencies
* softwarerero:accounts-t9n@1.1.4 <- useraccounts:core 1.12.4 <- useraccounts:materialize 1.12.4 <- my-app:external-dependencies 0.0.2
```

So I checked `useraccounts:core`. I found that they know this causes a bug, but they [haven't upgraded because](https://github.com/meteor-useraccounts/core/issues/571) 1.1.6 has [not been released](https://atmospherejs.com/softwarerero/accounts-t9n).

Please release the fix!

**Comment by thomasvanlankveld** [comment]
After git cloning into `/packages` I see it might not be ready yet...

You probably need the dependency `blaze-html-templates` instead of `blaze` for Meteor 1.2...

**Comment by softwarerero** [comment]
I see there is a problem. I will try to dig into soon.

**Comment by softwarerero** [comment]
I changed blaze to blaze-html-templates and published 1.1.6. @splendido: It should be good to update now.
